module1.js

```diff
@@ -51,10 +61,10 @@ function question3(number) {
   For example, if the input array is: [1,2,3,4,5,6]
   the output would be: [2,4,6]
 */
-
-function question4(array) {
-  // TODO:
-}
+var arr = [4,5,7,8,14,45,76];
+var filtered = arr.filter(function(element, index, array) {
+  return (index % 2 === 0);
+});
```

Comment:
This isn't actually inside the body of `question4` any more, you've deleted that function. Did you find this implementation somewhere? It works, but seems on a different level to the other code you've written here.